tests/tck/features/match/Unwind.feature

```diff
@@ -149,7 +149,7 @@ Feature: Unwind clause
     Then a SyntaxError should be raised at runtime: syntax error near `WITH'
     When executing query:
       """
-      MATCH (a:player {name:"Tim Duncan"}) - [e:like] -> (b)
+      MATCH (a:player {name:"Tim Duncan"})-[e:like]->(b)
```

Comment:
These are obviously unaccepted.

Reply:
I don't think it's a big deal.

Reply:
You also could write as this: MATCH (a:player {name:"Tim Duncan"}) -[e:like]-> (b)